front/src/common/authorization/components/InitialRedirect.tsx

```diff
@@ -0,0 +1,13 @@
+import { useSelector } from 'react-redux';
+import { Navigate } from 'react-router-dom';
+
+import Home from 'main/home';
+import { getIsStdcmProfile } from 'reducers/user/userSelectors';
+
+const InitialRedirect = () => {
+  const isStdcmProfile = useSelector(getIsStdcmProfile);
```

Comment:
This checks if the user has the `Stdcm` role. Wouldn't a super-user have all of the roles? In that case, I don't think they should be redirected?

Maybe we should instead check if the user only has the role to access STDCM, and has no permission to access any other tool?

Reply:
I agree. But I'm not sure it's possible to access stdcm without additional permissions like mapread for example. 
Then, as we'll need those permissions, the mapViewer module would be accessible in Home? Should we redirect the user to stdcm even if he could get access to some of the modules inside Home ?

Reply:
What about this: if the user has access to exactly all of the roles required for STDCM and nothing more, redirect to STDCM. If they have more permissions (e.g. operational studies), or less permissions (e.g. only map read), don't redirect.